### PR TITLE
Changes to release_notes_url and offering_docs_url

### PR DESCRIPTION
**Pull request by NatarajBTI**
Changes to release_notes_url and offering_docs_url

### CODE REVIEW

**Comment by NatarajBTI** [comment]
@ocofaigh - Please review & approve this PR..thanks

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I think `docs-draft` should be `docs` for production - can you confirm with your docs contact?

**Comment by NatarajBTI** [comment]
@ocofaigh - Please review this PR again as many things are updated & merge the same. I have difficulty in creating a symbolic link so have placed the override.json file in a folder.

**Review by ocofaigh — COMMENTED** [review]
@NatarajBTI Ideally your tests should be pointing to the override json that your docs are too. But it looks like you have just duplicated the file and added it to `override-json-file/override.json`. These will become out of sync, so I suggest you just keep 1 file. 
I believe the file has to exist in the same folder as the code that landing zone is called in (aka `tests/resources`) so how about maybe creating a symlink for `override-json-file/override.json` which just points to the one in the test folder to avoid duplication (or visa versa) and see if that works?

**Comment by ocofaigh** [comment]
Also @NatarajBTI please have your content team review the updated reference architecture markdown updates

**Review by Tejashree-Shiju — APPROVED** [review]
@NatarajBTI The architecture.md changes look good to me.

**Comment by NatarajBTI** [comment]
@ocofaigh - Have created symbolic link and Tejashree from ID team has approved the changes. Please re-review it. thanks

File: override-json-file
Comment (ocofaigh) [inline override-json-file:1]: @NatarajBTI I think you might need to symlink it the other way around, because if your docs is pointing to this file, they won't see the contents of the file. However if you added the override.json here and made `tests/resources/override.json` a symlink, then it should work

Comment (ocofaigh) [reply override-json-file:1]: Also in your last commit this file was in a directory, but now its in the root of the repo. Please confirm what your docs are going to point to (cc @Tejashree-Shiju )

Comment (NatarajBTI) [reply override-json-file:1]: > @NatarajBTI I think you might need to symlink it the other way around, because if your docs is pointing to this file, they won't see the contents of the file. However if you added the override.json here and made `tests/resources/override.json` a symlink, then it should work

ok..will try to do other way but I am not sure whether that will work & whether symbolic link to file will be picked up in main.tf file here --> https://github.com/terraform-ibm-modules/terraform-ibm-mas/blob/main/tests/resources/main.tf

Comment (NatarajBTI) [reply override-json-file:1]: > Also in your last commit this file was in a directory, but now its in the root of the repo. Please confirm what your docs are going to point to (cc @Tejashree-Shiju )

I am waiting for your approval & then will ask @Tejashree-Shiju to update the path to override.json file accordingly.

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@NatarajBTI you have incorrectly symlinked the override.json. You need to do this:

1. `cd tests/resources`
2. `ln -s ../../override-json-file/override.json override.json`

and commit

**Comment by ocofaigh** [comment]
I just updated for you, re-running..

File: tests/resources/main.tf
Comment (ocofaigh) [inline tests/resources/main.tf:11]: @NatarajBTI remove this now that we have the symlink working

File: tests/resources/main.tf
Comment (NatarajBTI) [reply tests/resources/main.tf]: It's still failing even after updating override_json_path to "${path.module}/../../override-json-file/override.json".
 Error: Invalid function argument on .terraform/modules/landing_zone/patterns/roks/module/config.tf line 50, in locals:
At this line of code -->
https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/b53a094273ec630951d7e74919dd7c15b12591ec/patterns/roks/module/config.tf#L50

Comment (NatarajBTI) [reply tests/resources/main.tf]: I think it should be "${path.root}/../../override-json-file/override.json" instead of "${path.module}/../../override-json-file/override.json"
Will try this..

**Review by ocofaigh — APPROVED** [review]

**Comment by NatarajBTI** [comment]
@ocofaigh - Have reverted back the path changes & our doc is pointing to the override.json file under {path.root}/override-json-file folder..Please review and merge the same..thanks

**Pull request merged**
